src/constants/tokenAddresses.ts

```diff
@@ -62,6 +62,9 @@ export const platform: Record<number, string> = {
   [137]: 'polygon-pos'
 }
 
+export const URL_KASSANDRA_CANDIDATE =
+  'https://www.kassandra.finance/ownership-claim'
```

Comment:
It should be the link to the app, and it could only have one variable URL_KASSANDRA